odoo/addons/base/models/ir_embedded_actions.py

```diff
@@ -20,7 +20,7 @@ class IrEmbeddedActions(models.Model):
     python_method = fields.Char(help="Python method returning an action")
 
     user_id = fields.Many2one('res.users', string="User", help="User specific embedded action. If empty, shared embedded action", ondelete="cascade")
-    is_deletable = fields.Boolean(compute="_compute_is_deletable")
+    is_deletable = fields.Boolean()
```

Comment:
I feel it won't be so easy like this.
When you change the field to `stored` you actually change the `defualt` logic for the field. When a `ir.embedded.actions` is created not when loading module data. Its `is_deletable` will be `False`. However, logically, it should be `True` since it has no external id.

Reply:
As far as i know, the only way to create a new embedded action is to use the 'save view' button from the top bar. When an action is created with it the 'is_deletable' field is set to True in the values, so even if i agree that it change the logic, i dont think it's gonna be a problem for the user, since there won't be any changes of behavior on his side. And removing that compute allow us to create the demo data with xml record instead of using _post_init to generate them with a manual call to create.